web_monitoring/tests/test_html_diff.py

```diff
@@ -89,6 +90,45 @@ def test_contrived_examples_htmldiffer(fn):
     return d
 
 
+def test_deactivate_deleted_active_elements():
```

Comment:
Would you please move this test to `test_html_diff_validity.py`? The tests in this module are meant to give us a set of real-world test cases for determining whether changes to the diff algorithm are better or worse, but not to test correctness (that’s why there are no assertions in this file).

Reply:
In fact, there’s already a similar test in that module: https://github.com/edgi-govdata-archiving/web-monitoring-processing/blob/fc36c7c0a902b71de372cc1bd721ee2b8f06fc30/web_monitoring/tests/test_html_diff_validity.py#L35-L66

So it might be a good template to follow for this.